src/sidebar/store/modules/viewer.js

```diff
@@ -27,12 +40,28 @@ function setShowHighlights(show) {
   return { type: actions.SET_HIGHLIGHTS_VISIBLE, visible: show };
 }
 
+/**
+ * @param {boolean} sidebarState - If the sidebar is open
+ */
+function setSidebarOpened(sidebarState) {
+  return { type: actions.SET_SIDEBAR_OPENED, sidebarState };
+}
+
+/** Selectors */
```

Comment:
`/** ... */` comments should be reserved for documentation of specific functions/variables etc. I have a feeling that tooling (such as VS Code) will think that this is the docstring for the `hasSidebarOpened` function otherwise.